config/locales/en.yml

```diff
@@ -339,6 +339,7 @@ en:
     search_heading: 'Publish a job'
     search_subheading: 'Which school is the job job for?'
     info: 'About %{school}'
+    map_aria_label: 'School location map'
```

Comment:
I believe the desired label is `Interactive map showing school location`, Michael suggested this yesterday evening